### DISCUSSION TITLE
Not getting any response from sk, curl command is stuck

### DISCUSSION THREAD

**Discussion by pomdtr** [johnlindquist/kit#364] [Errors] [answered]
I am not getting any response from kit when running the `sk` script:

```
/tmp $ echo 'await arg("Hi")' > foo.js
/tmp $ sk foo.js
{"script":"foo.js","args":[]}
```

Is there something wrong with my command ?

This also does not work on the built-in scripts :'(

**Comment by johnlindquist** [comment, marked as answer]
(We chatted about this in an issue, but answering here too)

If a `book-seach` script exists in your kit environment (kenv), then `sk` works like this:

```bash
~/.kit/bin/sk book-search
```

Adding `~/.kit/bin`to your `$PATH` lets you do:
```bash
sk book-search
```

## Running scripts from anywhere is broken 🤦‍♂️

I refactored `sk` into a unix socket a couple weeks back to replace the old, slower way of spinning up a node IPC channel, but looks like I forgot to add the "Run script from anywhere" logic back in. I'll get that out in a build in a couple of days. Your example should have worked.
